flink-read-write-hive/README.md

```diff
@@ -57,21 +56,28 @@ to run this example.
    $ docker exec -w /root/flink-read-write-hive flink bash -c "export HADOOP_CLASSPATH=\`hadoop classpath\`; python main.py"
    ```
 
-   You should be able to find the following information printed out in the end
-   of the terminal.
+   The program would create and submit two Flink jobs, the second of which runs
+   infinitely as the data stream read from Hive will not be regarded as finished.
```

Comment:
the second of which runs infinitely as the data stream read from Hive will not be regarded as finished
->

The second job reads Hive table as an unbounded stream and runs continuously until it is explicitly cancelled.